### PR TITLE
[FIX] Fix the error raised when build the image

### PR DESCRIPTION
**Pull request by ruiztulio**
Seems that a dependency is braking the install process of panama-dv and pylint-odoo, but I still can't find why and what package do that, but it seems that installing them first solve the issue by now.

@moylop260 Moving the dependencies to be installed before the requirements file seems to fix this by now, I'm still trying to find exactly what's the dependency that is braking the build because it is not panama-dv nor pylint-odoo.

I'll leave this here in case you want to use this as a workaround while I try to find the real fix for the error

### CODE REVIEW

File: odoo100/scripts/10.0-full_requirements.txt [hunk 1]
```diff
@@ -29,5 +29,3 @@ sphinx-patchqueue==0.4.0
 sphinx==1.2.3
 unidecode
 git+https://github.com/savoirfairelinux/num2words.git@0.5.4
```
Comment (moylop260) [inline odoo100/scripts/10.0-full_requirements.txt]: This is duplicated
Can we take advantage to remove it?

Comment (ruiztulio) [inline odoo100/scripts/10.0-full_requirements.txt]: @moylop260 num2words? I'm not able to find it, can you point me to the line where it is repeated please?

Comment (moylop260) [inline odoo100/scripts/10.0-full_requirements.txt]: line 20

**Comment by ruiztulio** [comment]
@moylop260 Done, also removed a duplicated dependency from the 8.0 file

**Comment by moylop260** [comment]
thank you

**Pull request merged**